kite-data/kite-data-core/src/main/java/org/kitesdk/data/Datasets.java

```diff
@@ -257,12 +310,18 @@
   /**
    * Update a {@link Dataset} for the given dataset or view URI.
    * <p>
-   * URI formats are defined by {@code Dataset} implementations, but must begin
-   * with "dataset:" or "view:".
+   * You can add, remove, or change the datatype of columns in your dataset,
+   * provided you don't attempt a change that would corrupt the data. Kite
+   * follows the Avro guidelines for updating schemas. See 
+   * <a href="http://kitesdk.org/docs/current/Schema-Evolution.html">Schema
```

Comment:
I don't think so. I can't find any syntax that allows it. Here's a link to the Javadoc spec.

http://docs.oracle.com/javase/1.5.0/docs/tooldocs/windows/javadoc.html#{@link}

@see doesn't help, either, because it creates a See Also: section. I want to keep the link under the URIs heading.

I spent a considerable amount of time chasing this down: if you can show a syntax example, please do.